wiki/release/release-process.md

```diff
@@ -467,11 +460,18 @@ cp pulsar-broker/target/docs/*.json site2/website/static/swagger/2.X.0
 ```
 Send out a PR request for review.
 
-## 16. Write release notes
+## 15. Write release notes
+
+Check the milestone in Github associated with the release.
+https://github.com/apache/pulsar/milestones?closed=1
+
+In the release item, add the list of most important changes that happened in the
+release and a link to the associated milestone, with the complete list of all the
+changes.
```

Comment:
I think we should just remove this part. There is no release item in the link.

Reply:
+1. I'll remove them